### PR TITLE
scaling factor in muon_integrator

### PR DESCRIPTION
**Pull request by AMWMitchell**
A second attempt at implementing the scaling factor change in muon_integrator. 
This time with commit history cleaned up. 

Linearity plots as before, further checks from other users welcome. 

sstlinearitytooltest

mstlstlinearitytooltest

### CODE REVIEW

File: ctapipe/image/muon/muon_integrator.py [hunk 1]
```diff
@@ -335,15 +334,16 @@ def calc_likelihood(image, pred, spe_width, ped):
...
-        ped = ped * u.deg
-        image = image * u.deg
-        sq = 1 / np.sqrt(2 * np.pi * (ped**2 + pred * (1 + spe_width**2) * u.deg))
+        #ped = ped * u.deg
+        #image = image * u.deg
+        pred = pred * u.deg
+        sq = 1 / np.sqrt(2 * np.pi * (ped**2 + pred * (1 + spe_width**2) ))#* u.deg))
         diff = (image - pred)**2
-        denom = 2 * (ped**2 + pred * (1 + spe_width**2) * u.deg)
+        denom = 2 * (ped**2 + pred * (1 + spe_width**2) )#* u.deg)
```
Comment (kosack) [inline ctapipe/image/muon/muon_integrator.py]: remove these commented out bits before merging

File: ctapipe/image/muon/muon_integrator.py [hunk 2]
```diff
@@ -335,15 +334,16 @@ def calc_likelihood(image, pred, spe_width, ped):
...
         expo = np.exp(-diff / denom) * u.m
         sm = expo < 1e-300 * u.m
         expo[sm] = 1e-300 * u.m
-        log_value = sq * expo / u.m * u.deg
+        log_value = sq * expo / u.m #* u.deg
```
Comment (kosack) [inline ctapipe/image/muon/muon_integrator.py]: remove `# * udeg` if it's incorrect

**Comment by AMWMitchell** [comment]
Yes - there was a scaling factor used in the HESS code that was missing, accounting for the pixel size, i.e. how many pixels are expected to lie along the ring circumference/azimuthal direction. The "image_prediction" function already accounts for the pixel width in the radial direction. 

This is probably at least part of the normalisation problem, but might not be the full solution / final answer. 
Note: fine tuning could account for true pixel geometries (assumed circular here) / gaps between pixels in the prediction.

**Comment by kosack** [comment]
This commit seems to have a lot more changes in it than mentioned (like the code in leakage.py).  I'm not sure why that is since that code is already merged in - can you perhaps make sure you have done a `git pull upstream master; git push` to ensure you have your git repo up to date? I suspect your branched repo is somehow out of sync with the upstream CTA one

**Comment by AMWMitchell** [comment]
Just tried that "git pull upstream master" then "git push" - doesn't seem to have changed things, unfortunately. It was only one file with changes; after Ruben wrote, I deleted the muon_fix branch thinking this commit had already been merged. When I realised my mistake, I tried to recreate the branch again which seems to have caused a mess. Maybe there's someway to revert to the version I wanted to commit?

**Comment by MaxNoe** [comment]
I don't know what happened here. But you seem to do weird things when you create new branches.

When you create a new branch, make sure you create it from the current upstream master:
```
git fetch upstream
git checkout -b <new_branchname> upstream/master
```

To resolve this, I would propose creating a new branch and cherry-picking your commit. 

```
git fetch upstream
git checkout -b muon_fixes_clean upstream/master
git cherry-pick dcb9eac 
git push -u origin muon_fixes_clean
```

And open a new PR.

Make sure, your master is identical to cta-observatory:master and always branch from `upstream/master` after `git fetch upstream`

**Pull request merged**